### PR TITLE
Added Dockey 1.0

### PR DESCRIPTION
**Pull request by bxio**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/dockey.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version :latest
+  sha256 :no_check
+
+  url 'http://dockey.publicspace.co/download/Dockey-latest.zip'
+  name 'dockey'
+  homepage 'http://dockey.publicspace.co/'
+
+  app 'dockey.app'
+
+  uninstall quit: 'com.igorstumberger.dockey'
```
Comment (adidalal) [inline Casks/dockey.rb]: Unneeded, we handle quitting the app automatically.

File: Casks/dockey.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'dockey' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://dockey.publicspace.co/download/Dockey-latest.zip'
+  name 'dockey'
+  homepage 'http://dockey.publicspace.co/'
+
+  app 'dockey.app'
+
```
Comment (adidalal) [inline Casks/dockey.rb]: You should add the minimum version as a requirement.
> Compatible with macOS 10.13 and newer 

would become: `depends_on :macos => '>= :high_sierra'`

**Review by adidalal — CHANGES_REQUESTED** [review]

File: Casks/dockey.rb (hunk 1 above)
Comment (commitay) [reply Casks/dockey.rb]: > we handle quitting the app automatically

How?

Comment (adidalal) [reply Casks/dockey.rb]: Isn't that https://github.com/Homebrew/brew/blob/master/Library/Homebrew/cask/lib/hbc/artifact/abstract_uninstall.rb#L116

Comment (adidalal) [reply Casks/dockey.rb]: Nvm, just reread the code, this is fine to have.

Comment (commitay) [reply Casks/dockey.rb]: It isn't automatic, still needs the bundle id.

File: Casks/dockey.rb (hunk 2 above)
Comment (bxio) [reply Casks/dockey.rb]: Thanks, added.

**Review by adidalal — APPROVED** [review]

**Pull request merged**